.travis.yml

```diff
@@ -1,7 +1,5 @@
 language: ruby
 rvm:
-- 2.0.0
-- '2.1'
 - '2.2'
 - '2.3'
```

Comment:
hi guys, this is not related to the changes in this PR, but just wanted to highlight that [Ruby versions up to 2.4 has already reached EOL.](https://www.ruby-lang.org/en/downloads/branches/)
I'm guessing clients using this SDK are probably / hopefully on Ruby 2.5 and above at least so it'll be nice to add these versions in the matrix to confirm compatibility. Just my 2 cents / yen worth 🤓

Reply:
@mt-kelvintaywl thank you for your advice! I'll add newer versions and try to run travis again :+1: